### ISSUE TITLE
Display Mounted Volumes

### ISSUE THREAD

**Issue by InklingGirl** [home-sweet-gnome/dash-to-panel#1888]
I think being able to display display mounted volumes (e.g. internal drives &/or USB-attached storage) would perfect `dash-to-panel`, so please consider it! Thank you.

**Comment by RichardJECooke** [comment]
Yes, please add this - umounted drives too. Basically the functionality that the standard Gnome dash has should still be available.

My NTFS drive never mounts at startup, despite being set to `automount` in Disks. So I have to click it on the taskbar whenever booting so various apps can see files inside.

**Comment by InklingGirl** [comment]
If you (safely) mess around w/ `fstab` you could probably get that drive successfully auto-mounting, but yes, `dash-to-panel` really should have this feature as it's a very convenient on other docks.

**Comment by InklingGirl** [comment]
nah ain't gonna let that happen!

**Comment by bazjapan** [comment]
+1

**Comment by InklingGirl** [comment]
I hope the developer will consider this & fork/port over the code that enables this from the other dock extensions supporting it.

**Comment by InklingGirl** [comment]
keeping the dream alive 🙏

**Comment by InklingGirl** [comment]
NOPE

**Comment by InklingGirl** [comment]
nah

**Comment by InklingGirl** [comment]
nooo!!
